src/pages/idea-market/IdeaMarketMain.tsx

```diff
@@ -61,19 +232,31 @@ export const IdeaMarketMain = () => {
           </div>
           <Carousel
             buttonPosition='center'
-            cardWidth={200}
+            cardWidth={200} // 🔹 카드 전체 크기 조정
             cardCount={3}
             gap={45}
-            dataLength={8}>
-            {cardsData.map((card) => (
-              <Card
-                key={card.id}
-                id={card.id}
-                isBookmarked={card.isBookmarked}
-                saves={card.saves}
-                views={card.views}
-                onBookmarkClick={handleBookmarkClick}
-              />
+            dataLength={ideaData.length} // 🔹 `ideaData` 그대로 활용
+          >
+            {ideaData.map((idea) => (
+              <div
+                key={idea.ideaId}
+                className={styles.carouselItem}>
+                <PreviewThumbnail
+                  ideaId={idea.ideaId}
+                  username={idea.writerName}
+                  description={idea.title}
+                  price={idea.price}
+                  imageUrl={idea.thumbnailImageUrl || ''}
+                  profileImage={idea.writerImageUrl}
+                  isBookmarked={idea.isSavedPost}
+                  saves={idea.saveCount}
+                  views={idea.viewCount}
+                  auth={idea.auth}
+                  category={idea.category}
+                  size='large'
```

Comment:
props를 객체로 변경하면 하나하나 내릴 필요 없어요!

```suggestion
                <PreviewThumbnail
                  ideaData = {idea}
                  size='large'
```